sys/src/libc/port/errno2str.c

```diff
@@ -0,0 +1,155 @@
+/*
+ * This file is part of the UCB release of Plan 9. It is subject to the license
+ * terms in the LICENSE file found in the top-level directory of this
+ * distribution and at http://akaros.cs.berkeley.edu/files/Plan9License. No
+ * part of the UCB release of Plan 9, including this file, may be copied,
+ * modified, propagated, or distributed except according to the terms contained
+ * in the LICENSE file.
+ */
+
+#include <u.h>
+#include <libc.h>
+
+static char *errors[] = {
+	[1] = "operation not permitted",
+	[2] = "no such file or directory",
```

Comment:
What system are these error strings from? Maybe we should match them to what Go expects on Plan 9:
```
$ go doc -all syscall | grep ENOENT
	ENOENT          = Errno(0x2)
$ GOOS=plan9 go doc -all syscall | grep ENOENT
	ENOENT       = NewError("file does not exist")
```

Reply:
They are from the p9 server. 
I believe they come from Linux directly, which I figure is OK. This is definitely the unix text. 

What really ought to happen is we use the constants in libc. I'll take a look.

Reply:
OK, see what you think, I lined them up I think.